docs/operations/cli.rst

```diff
@@ -117,10 +117,19 @@ following are the command line options that Envoy supports.
   during a hot restart. See the :ref:`hot restart overview <arch_overview_hot_restart>` for more
   information. Defaults to 900 seconds (15 minutes).
 
-.. option:: --max-stat-name-len <uint64_t>
+.. _operations_cli_max_obj_name_len:
```

Comment:
doc nit: You don't need this anchor, you can reference options directly and then they look nice inline. See how we ref other options.